pkg/services/app/add.go

```diff
@@ -132,6 +132,23 @@ func (a *App) Add(params AddParams) error {
 	}
 }
 
+func getAppHash(params AddParams) (string, error) {
+	var appHash string
+	var err error
+	if DeploymentType(params.DeploymentType) == DeployTypeHelm {
+		appHash, err = utils.GetHash(params.Url, params.Chart, params.Branch)
```

Comment:
I would argue that this `GetHash` function isn't providing any abstraction anymore; it is just wrapping the native `go` libraries to create a content hash. 

It also doesn't get called anywhere but this package (as far as I can tell), so I would move that code here to avoid unnecessary indirection.

Reply:
yes, I agree, let's fix that as well